### PR TITLE
Update uptime.markdown

### PR DESCRIPTION
**Pull request by faster777**
## Proposed change

Update description of Uptime integration to reflect latest changes and current state; Uptime now represents the timestamp of last startup, and not time since last restart.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @faster777, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16461).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by faster777** [comment]
@chemelli74 please review this quick doc change for Uptime integration.  Thanks.

**Comment by frenck** [comment]
@faster777 Please don't tag random people to ask for reviews 👍

File: source/_integrations/uptime.markdown [hunk 1]
```diff
@@ -10,7 +10,9 @@ ha_quality_scale: internal
 ha_domain: uptime
 ---
 
-The `uptime` sensor platform displays the time since the last Home Assistant restart.
+Starting with release 2020.12 the `uptime` sensor platform displays the timestamp (date and time) when Home Assistant was last started.
```
Comment (frenck) [inline source/_integrations/uptime.markdown:13]: We don't log changes in our documentation. Our documentation reflects our current state of things matching our current stable release.

For changes, please see the release notes.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing this PR, as it only contains a changelog, which we don't document in the documentation.
Nevertheless, thanks for contributing 👍

**Comment by faster777** [comment]
Thanks for the feedback @frenck .  I am new to github and the HA community, so I thought I would start with a simple doc-only change...

First, I was not attempting to tag random people; chemelli74 initiated the original PR #43623 back in Nov/Dec timeframe that introduced the breaking change for Uptime integration.  However the documentation page for that integration still describes it the old way.  Hence I tagged him for review to make sure this doc-only change also matches that PR.

Second, I can remove the historical reference to prior behavior of Uptime integration, and only mention the current behavior.  I still think that is a valid update to help other users of this Integration.  Thanks.

**Pull request closed without merging**